R/plot_epiparameter_distribution.R

```diff
@@ -0,0 +1,56 @@
+#' Plot an epiparameter distribution object
+#'
+#' @description Produces a simple plot, using base R, of an epiparameter
+#' distribution object of the class [epiparameter::epidist()].
+#'
+#' @param epidist An epiparameter object retrieved using the function
+#' [epiparameter::epidist_db()] or otherwise constructed using
+#' [epiparameter::epidist()]
+#'
+#' @param from The day from which to start plotting the distribution. Default
+#' option is day 0.
+#'
+#' @param to The day to which to start plotting the distribution. Default
+#' option is day 30.
+#'
+#' @param by The time resolution at which which to plot the distribution.
+#' Default option is 0.1 days.
+#'
+#' @return A plot of the delay distribution over time
+#'
+#' @export
+#'
+#' @examples
+#' # Retrieve the delay distribution between onset and death for Ebola,
+#' # using the epiparameter function `epiparameter::epidist_db()`
+#'
+#' onset_to_death_ebola <- epiparameter::epidist_db(
+#'   disease = "Ebola Virus Disease",
+#'   epi_dist = "onset_to_death",
+#'   author = "Barry_etal"
+#' )
+#'
+#' plot_epiparameter_distribution(onset_to_death_ebola, by = 1)
+plot_epiparameter_distribution <- function(epidist,
+                                           from = 0,
+                                           to = 30,
+                                           by = 0.1) {
```

Comment:
I think {epiparameter} has a `plot()` method for `epidist` objects - I wonder whether this function can then be removed. If this is a distinct method that achieves something quite different, it might be worth formally making this an S3 method for `epidist`s.